lib/rules/global-require.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * global-require
+ */
+
+'use strict';
+
+var ACCEPTABLE_PARENTS = [
+    'AssignmentExpression',
+    'VariableDeclarator',
+    'MemberExpression',
+    'ExpressionStatement',
+    'Program',
+    'VariableDeclaration'
+];
+
+module.exports = function(context) {
+    return {
+        "CallExpression": function(node) {
+            if (node.callee.name === "require") {
+                var isGoodRequire = context.getAncestors().every(function(x) {
+                    return ACCEPTABLE_PARENTS.indexOf(x.type) > -1;
+                });
+                 if (!isGoodRequire) {
+                        context.report(node, "Unexpected require() inside a block statement");
```

Comment:
Hmm, maybe just "Unexpected require()." is good enough? It might not always be in a block statement, it might be in a function too.